pdns/recpacketcache.cc

```diff
@@ -161,6 +165,7 @@ void RecursorPacketCache::insertResponsePacket(unsigned int tag, uint32_t qhash,
     }
```

Comment:
We need to account for the real size of the protobuf message somehow :-/

Reply:
And for the size of the indexes, see below.

Reply:
> We need to account for the real size of the protobuf message somehow :-/

The python protobuf implementation keeps a running count of the (wire!) size. I haven't spotted such a thing in the C++ lib, but it would be too low anyway. I wonder if just assuming it doubles the size of the whole cache entry is close enough.